backend/franky/model/project.py

```diff
@@ -3,16 +3,22 @@
 
 class ProjectData:
 
-    def __init__(self, name: str, start: str, end: Optional[str], tags: List[str]):
+    def __init__(self, name: str, start: Optional[str], end: Optional[str], tags: List[str],
+                 url: str, size: float, pull_count: int, star_count: int):
         """
         User project data.
 
         :param name: Project name.
         :param start: Project start date.
         :param end: Project end date.
+        :param url: Url to project
         :param tags: A list of tags that project is associated with.
         """
         self.name = name
         self.start = start
         self.end = end
+        self.url = url
         self.tags = tags or []
+        self.size = size or None
+        self.pull_count = pull_count or None
+        self.star_count = star_count or None
```

Comment:
Types in python are used to declare contracts. In this case both `size` and `star_count` fields is of `int` type. Therefore they cannot be assign with `None` value.

There are two possibilities - you can remove `or None` expressions or you can use `Optional[int]` as the fields type.